docs/src/using-plugins.md

```diff
@@ -144,7 +150,9 @@ flowchart TB
         --> gen_C(buildEnd):::parallel
     end
 
-    A[start] --> B(resolveConfig) --> generate  --> E[write files]
+    subgraph &nbsp;
```

Comment:
I don't know if it's worth making this a subgraph, if there is no name.

I'd just make this the main graph, no?

Reply:
I moved it to subgraph as the look is better ([check it here](https://mermaid.live/edit#pako:eNp1kstqwzAQRX9FqFAciDfNToVCnEc33bW7KBRZGtkisuTq0VBC_r2yjeMmobMayWfuvR77hLkVgAnO85yaoIIGgl7BgGNBWYNqaw-emv6p1PbIa-YC-iioQam4Zt6vQaKWOaY1aCSV1uRhuy0WxWLOrbaOlJrxww3v4StCsmPTxPrpdmKY8bGsHGtr9AYVGDFcdjV6ZmMzI4Rcgkzc5JVNbcf-CTHQSR7d2VbDMmASFMoB77cz7qGrxH0uMx_SfoqotPgnTZ6_9GiRBceMl9Y111nu0VVWdoIbc6V5iXyX-NGUvn2edJa7PtW-FywyB97qb1hZI1U1G136dxw8N7ujU-mQvgz4_cUHz3EDrmFKpN_l1F1THGpogGKSWgGSRR0opuacUBaDff8xHJPgIsyxs7GqMZFM-3SKrUh2a8VS4mZAzr_I4dNS)). Also, removed the group title as we already have the whole diagram title. But we can give it a title too. any suggestion?